CalendarApp/BookingRoom/Controllers/DeletingController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using BookingRoom.Models;
+
+namespace BookingRoom.Controllers
+{
+    public class DeletingController : ApiController
```

Comment:
Rename to DeleteController. Same for the rest controllers
Object naming has to be consistent